yjit/src/codegen.rs

```diff
@@ -298,6 +298,16 @@ impl JITState {
             }
         }
     }
+
+    /// Return true if we're compiling a send-like instruction, not an opt_* instruction.
+    pub fn on_send_insn(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_send_insn(&self) -> bool {
```

Maybe `is_send` or `is_sendish` ? 😅

Reply:
chose `is_sendish` https://github.com/ruby/ruby/pull/10033/commits/761fa5e22856cff1f1bb8f60ecf46b7ea13e3c87